.asf.yaml

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: World's most powerful open data catalog for building a high-performance, geo-distributed and federated metadata lake.
+  homepage: https://datastrato.ai/docs/
+  labels:
+    - metadata
+    - data-catalog
+    - datalake
+    - stratosphere
+    - federated-query
+    - lakehouse
+    - model-catalog
+    - metalake
+    - skycomputing
+    - ai-catalog
+    - opendatacatalog
+  features:
+    # Enable wiki for documentation
+    wiki: true
```

Comment:
We don't need to enable wiki, we will use website for documentation.

Reply:
I changed it to false.